templates/cloudformation/openshift_account_roles.json

```
@@ -0,0 +1,347 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Parameters" : {
+      "RedHatAWSAccountID": {
+        "Description": "AWS Account ID",
+        "Type": "Number",
+        "Default": "710019948333"
+      },
+      "ROSAVersion": {
+        "Description": "ROSA version",
+        "Type": "Number"
+      },
+      "Prefix": {
+        "Description": "Role and Policy name prefix",
+        "Type": "String",
+        "AllowedPattern": "^[a-zA-Z0-9-_]*$",
+        "MaxLength": 32,
+        "Default": ""
+      }
+    },
+    "Resources": {
+        "RHManagedOpenShiftInstallerRole": {
+            "Type": "AWS::IAM::Role",
+            "Properties": {
+                "RoleName": { "Fn::Join" : ["", [ { "Ref": "Prefix" }, "ManagedOpenShift-Installer-Role" ]] },
```

I think these lines should be
```suggestion
                "RoleName": { "Fn::Join" : ["", [ { "Ref": "Prefix" }, "-Installer-Role" ]] },
```
and the `Prefix` is set by the user (or defaults to `ManagedOpenShift`)